bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/robot/RRMapDraw.java

```diff
@@ -331,19 +331,23 @@ private void drawOpenHabRocks(Graphics2D g2d, int width, int height, float scale
         } catch (IOException e) {
             logger.debug("Error loading image ohlogo.png:: {}", e.getMessage());
         }
-        Font font = new Font("Helvetica", Font.BOLD, 14);
-        g2d.setFont(font);
-        String message = "Openhab rocks your Xiaomi vacuum!";
-        FontMetrics fontMetrics = g2d.getFontMetrics();
-        int stringWidth = fontMetrics.stringWidth(message);
-        if ((stringWidth + textPos) > rmfp.getImgWidth() * scale) {
-            font = new Font("Helvetica ", Font.BOLD,
-                    (int) Math.floor(14 * (rmfp.getImgWidth() * scale - textPos - offset * scale) / stringWidth));
+        try {
+            Font font = new Font("Helvetica", Font.BOLD, 14);
             g2d.setFont(font);
+            String message = "Openhab rocks your Xiaomi vacuum!";
+            FontMetrics fontMetrics = g2d.getFontMetrics();
+            int stringWidth = fontMetrics.stringWidth(message);
+            if ((stringWidth + textPos) > rmfp.getImgWidth() * scale) {
+                font = new Font("Helvetica ", Font.BOLD,
+                        (int) Math.floor(14 * (rmfp.getImgWidth() * scale - textPos - offset * scale) / stringWidth));
+                g2d.setFont(font);
+            }
+            int stringHeight = fontMetrics.getAscent();
+            g2d.setPaint(Color.white);
+            g2d.drawString(message, textPos, height - offset * scale - stringHeight / 2);
+        } catch (Exception e) {
```

Comment:
Please do not catch Exception. What Exception can occur here?

Reply:
@J-N-K see my first post on why I catch Exception. (No other Exception can be fought it seems to me)

Nevertheless as suggested I'll try to look for list of available fonts in the hope that that will solve the issue this user is having ( which may be openjdk issue as well)